### ISSUE TITLE
Bug

### ISSUE THREAD

**Issue by KSKOP69** [pytgcalls/pytgcalls#189]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Environment**
 - OS: [Linux]
 - Branch [master]
 - NTgCalls Version [v1.0.0.dev20]
 - Python Version [3.10.12]

**Additional context**

```
[13-Nov-23 17:35:22 - ERROR] - asyncio - Task exception was never retrieved
future: <Task finished name='Task-3333990' coro=<Start.start..participants_handler() done, defined at /usr/local/lib/python3.10/dist-packages/pytgcalls/methods/utilities/start.py:19> exception=ConnectionNotFound('Connection with chat id "-1001692328939" not found')>
Traceback (most recent call last):
File "/usr/local/lib/python3.10/dist-packages/pytgcalls/methods/utilities/start.py", line 34, in participants_handler
self._binding.get_state(chat_id),
ntgcalls.ConnectionNotFound: Connection with chat id "-1001692328939" not found
```

**Comment by KSKOP69** [comment]
```
[16-Nov-23 03:25:31 - WARNING] - asyncio - socket.send() raised exception.
[16-Nov-23 03:25:31 - WARNING] - asyncio - socket.send() raised exception.
[16-Nov-23 03:25:31 - WARNING] - asyncio - socket.send() raised exception.
[16-Nov-23 03:25:31 - WARNING] - asyncio - socket.send() raised exception.
[16-Nov-23 03:25:31 - WARNING] - asyncio - socket.send() raised exception.
[16-Nov-23 03:25:34 - WARNING] - asyncio - socket.send() raised exception.
[16-Nov-23 03:25:34 - WARNING] - asyncio - socket.send() raised exception.
[16-Nov-23 03:25:34 - WARNING] - asyncio - socket.send() raised exception.
[16-Nov-23 03:25:34 - WARNING] - asyncio - socket.send() raised exception.
[16-Nov-23 03:25:34 - WARNING] - asyncio - socket.send() raised exception.
[16-Nov-23 03:25:34 - WARNING] - asyncio - socket.send() raised exception.
[16-Nov-23 03:25:34 - WARNING] - asyncio - socket.send() raised exception.
[16-Nov-23 03:25:34 - WARNING] - asyncio - socket.send() raised exception.
[16-Nov-23 03:25:34 - WARNING] - asyncio - socket.send() raised exception.
[16-Nov-23 03:25:38 - WARNING] - asyncio - socket.send() raised exception.
[16-Nov-23 03:25:38 - WARNING] - asyncio - socket.send() raised exception.
[16-Nov-23 03:25:38 - WARNING] - asyncio - socket.send() raised exception.
[16-Nov-23 03:25:38 - WARNING] - asyncio - socket.send() raised exception.
[16-Nov-23 03:25:39 - WARNING] - asyncio - socket.send() raised exception.
[16-Nov-23 03:25:39 - WARNING] - asyncio - socket.send() raised exception.
[16-Nov-23 03:25:39 - WARNING] - asyncio - socket.send() raised exception.
[16-Nov-23 03:25:39 - WARNING] - asyncio - socket.send() raised exception.
[16-Nov-23 03:25:39 - WARNING] - asyncio - socket.send() raised exception.
[16-Nov-23 11:22:00 - ERROR] - asyncio - Task exception was never retrieved
future: <Task finished name='Task-1407343' coro=<Start.start.<locals>.participants_handler() done, defined at /opt/venv/lib/python3.10/site-packages/pytgcalls/methods/utilities/start.py:19> exception=ConnectionNotFound('Connection with chat id "-1001692328939" not found')>
Traceback (most recent call last):
  File "/opt/venv/lib/python3.10/site-packages/pytgcalls/methods/utilities/start.py", line 34, in participants_handler
    self._binding.get_state(chat_id),
ntgcalls.ConnectionNotFound: Connection with chat id "-1001692328939" not found
[16-Nov-23 11:22:28 - ERROR] - asyncio - Task exception was never retrieved
future: <Task finished name='Task-1408181' coro=<Start.start.<locals>.participants_handler() done, defined at /opt/venv/lib/python3.10/site-packages/pytgcalls/methods/utilities/start.py:19> exception=ConnectionNotFound('Connection with chat id "-1001692328939" not found')>
Traceback (most recent call last):
  File "/opt/venv/lib/python3.10/site-packages/pytgcalls/methods/utilities/start.py", line 34, in participants_handler
    self._binding.get_state(chat_id),
ntgcalls.ConnectionNotFound: Connection with chat id "-1001692328939" not found
[16-Nov-23 11:22:45 - ERROR] - asyncio - Task exception was never retrieved
future: <Task finished name='Task-1408678' coro=<Start.start.<locals>.participants_handler() done, defined at /opt/venv/lib/python3.10/site-packages/pytgcalls/methods/utilities/start.py:19> exception=ConnectionNotFound('Connection with chat id "-1001692328939" not found')>
Traceback (most recent call last):
  File "/opt/venv/lib/python3.10/site-packages/pytgcalls/methods/utilities/start.py", line 34, in participants_handler
    self._binding.get_state(chat_id),
ntgcalls.ConnectionNotFound: Connection with chat id "-1001692328939" not found
[16-Nov-23 11:22:55 - ERROR] - asyncio - Task exception was never retrieved
future: <Task finished name='Task-1408971' coro=<Start.start.<locals>.participants_handler() done, defined at /opt/venv/lib/python3.10/site-packages/pytgcalls/methods/utilities/start.py:19> exception=ConnectionNotFound('Connection with chat id "-1001692328939" not found')>
Traceback (most recent call last):
  File "/opt/venv/lib/python3.10/site-packages/pytgcalls/methods/utilities/start.py", line 34, in participants_handler
    self._binding.get_state(chat_id),
ntgcalls.ConnectionNotFound: Connection with chat id "-1001692328939" not found
[16-Nov-23 11:23:05 - ERROR] - asyncio - Task exception was never retrieved
future: <Task finished name='Task-1409299' coro=<Start.start.<locals>.participants_handler() done, defined at /opt/venv/lib/python3.10/site-packages/pytgcalls/methods/utilities/start.py:19> exception=ConnectionNotFound('Connection with chat id "-1001692328939" not found')>
Traceback (most recent call last):
  File "/opt/venv/lib/python3.10/site-packages/pytgcalls/methods/utilities/start.py", line 34, in participants_handler
    self._binding.get_state(chat_id),
ntgcalls.ConnectionNotFound: Connection with chat id "-1001692328939" not found
[16-Nov-23 11:23:24 - ERROR] - asyncio - Task exception was never retrieved
future: <Task finished name='Task-1409938' coro=<Start.start.<locals>.participants_handler() done, defined at /opt/venv/lib/python3.10/site-packages/pytgcalls/methods/utilities/start.py:19> exception=ConnectionNotFound('Connection with chat id "-1001692328939" not found')>
Traceback (most recent call last):
  File "/opt/venv/lib/python3.10/site-packages/pytgcalls/methods/utilities/start.py", line 34, in participants_handler
    self._binding.get_state(chat_id),
ntgcalls.ConnectionNotFound: Connection with chat id "-1001692328939" not found
```

**Comment by TuriOG** [comment]
Fixed on 1.0.2, try to update your environment and let us know if you face that issue again.
